gmp/gmp.py

```diff
@@ -1263,10 +1265,121 @@ def modifyTagCommand(self, tag_id, kwargs):
                ''.format(tag_id, name, resource, value, comment, active)
 
     def modifyTargetCommand(self, target_id, kwargs):
-        raise NotImplementedError
 
-    def modifyTaskCommand(self):
-        raise NotImplementedError
+        assert target_id
```

Comment:
Please do not use `assert` to enforce interface constraints, see #32.

The following would be better:
```
if not target_id:
    raise ValueError('modify_target requires a target_id element')
```